src/init/handlers.ts

```diff
@@ -9,7 +9,7 @@ import { hideMenu } from 'store/widgets/actions';
 import { outsideArticleClasses } from 'class_consts';
 import highlights from './highlights';
 
-let handlers;
+export let handlers;
```

Comment:
this probably doesn't need to get exported now?

Reply:
:+1: